### ISSUE TITLE
Select appearance columns-pack should not take width of labels into account

### ISSUE THREAD

**Issue by lognaturel** [getodk/collect#3744]
#### Software and hardware versions 
Collect v1.26.1

#### Problem description
Select appearance columns-pack should not take width of labels into account

#### Steps to reproduce the problem
Load a form with selects with the `columns-pack` appearance and choices of different lengths. [columns-pack.zip](https://github.com/opendatakit/collect/files/4378966/columns-pack.zip)

![device-2020-03-24-211201](https://user-images.githubusercontent.com/967540/77501115-396bf880-6e14-11ea-8734-68293c74eb5d.png)

#### Expected behavior
The padding between options should be fixed regardless of the label or image sizes. The current behavior is problematic when stacking choice lists where one is a subset of the other such as `Yes No Unknown` and `Yes No` in the form at https://forum.opendatakit.org/t/who-covid-19-minimum-reporting-form/25647

I think we should also add a little bit of extra space to the current padding when all choice labels are the same length. I think the very minimal padding works well with images but is too packed for text labels. Perhaps those could be two different cases?

**Comment by grzesiek2010** [comment]
So you would expect having cells with different sizes? Currently we calculate the longest one and it's the width for all items. 
It's not a new behavior for `columns-pack` it has worked in the same way for `compact`.

**Comment by lognaturel** [comment]
Historically, `compact` did not show radiobuttons/checkboxes and was really only practical for use with images. When we renamed the appearances, we introduced `columns-pack` with buttons for text options.

The nice thing about having the columns be the same size is that it lays out reasonably when there's wrapping at the screen boundary. With images in particular, you may want to have a big grid of several images and they may not have exactly the same size. I imagine that's why it was done that way initially.

With text, I don't think it's as compelling. `columns-n` is likely a better choice.

I see two options:
1. Instead of computing column widths for images and text, put a fixed amount of padding between choices. This would match what Enketo does:
<img width="373" alt="Screen Shot 2020-04-23 at 2 15 00 PM" src="https://user-images.githubusercontent.com/967540/80150231-dc777580-856c-11ea-95d2-004bea3e264c.png">

2. Keep the existing column width computation for images but put fixed padding for text.

I prefer the first option. It is a change in behavior and could possibly be problematic but it wouldn't be fatal. If users had a compelling use case we could move towards the second option.

I realize that we're not dealing with CSS so I'm not sure how difficult it would be to maintain even padding between choices rather than even column width. What are you thinking, @grzesiek2010?

**Comment by grzesiek2010** [comment]
Ok I understand thanks for your explanation. I'll take a look at possible solutions.

**Comment by grzesiek2010** [comment]
I investigated the issue and possible solutions. Looks like using RecyclerView + [FlexboxLayout](https://github.com/google/flexbox-layout) would be a great improvement. I created a sample app for testing and here is a screenshot:
![Screenshot_2020-05-08-11-34-15](https://user-images.githubusercontent.com/3276264/81387993-ecb85480-9117-11ea-9b01-48461932f4c0.png)

@lognaturel let me know if it's what you expected and if I can go ahead.
